deep-learning/_posts/2017-05-17-tiramisu.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: review
+title:  "The One Hundred Layers Tiramisu: Fully Convolutional DenseNets for Semantic Segmentation"
+tags:   deep-learning CNN segmentation
+author: Clément Zotti
+pdf:   "https://arxiv.org/pdf/1611.09326.pdf"
+
+cite:
+  authors: "S.Jégou, M.Drozdzal, D.Vasquez, A.Romero, Y.Bengio"
+  title:   "The One Hundred Layers Tiramisu: Fully Convolutional DenseNets for Semantic Segmentation"
+  venue:   "arXiv 2016"
+---
+
+This papers implement all the well known advanced in the image segmentation community and make it work together.
+
+List of the methods used:
+
+* Dense blocks
+* U-Net architecture
+* Using IOU loss
```

Comment:
The loss seems to be only cross_entropy : https://github.com/SimJeg/FC-DenseNet/blob/master/train.py#L101